File: server/src/main/java/com/uiptv/model/PublishedM3uSelection.java
package com.uiptv.model;

import com.uiptv.shared.BaseJson;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@EqualsAndHashCode(callSuper = false)
public class PublishedM3uSelection extends BaseJson {
    private String dbId;
    private String accountId;

    public PublishedM3uSelection(String accountId) {
        this.accountId = accountId;
    }
}

File: server/src/main/kotlin/com/uiptv/model/PublishedM3uSelection.kt
package com.uiptv.model

import com.uiptv.shared.BaseJson

class PublishedM3uSelection() : BaseJson() {
    var dbId: String? = null
    var accountId: String? = null

    constructor(accountId: String?) : this() {
        this.accountId = accountId
    }
}
